crypto/hash/hmac_nss.c

```diff
@@ -42,6 +42,9 @@
 #include "alloc.h"
 #include "err.h" /* for srtp_debug */
 #include "auth_test_cases.h"
+
+#define NSS_PKCS11_2_0_COMPAT 1
```

Comment:
Do we need this flag outside of the GCM case?

Reply:
likely not i was just being consistent with setting it before all includes